### PR TITLE
Web application site fix (Part - 30) Part 2

### PR DESCRIPTION
**Pull request by omar-srbd**
This PR includes the fix of the following files which are part of the web application guides section of the Tizen Docs site:

File count: 1

/application/web/guides/w3c/supplement/typedarray.md

### CODE REVIEW

File: docs/application/web/guides/w3c/supplement/typedarray.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ A typed array is capable of handling binary data in different sources:
 
 Additionally, a typed array can be used to manage in-memory binary data of byte layouts, and assist in efficiently manipulating raw data for complex and powerful Web applications, such as audio and video operation, Web sockets, and the [File API operations](../../data/file-system.md).
 
-## Creating an Array Buffer
+## Create an array Buffer
```
Comment (maliha-srbd) [inline docs/application/web/guides/w3c/supplement/typedarray.md:31]: ## Create an Array Buffer

File: docs/application/web/guides/w3c/supplement/typedarray.md [hunk 2]
```diff
@@ -137,10 +137,10 @@ To handle binary data in your application with JavaScript, you must learn to acc
    </script>
    ```
 
-	> **Note**  
+	> [!NOTE]
```
Comment (maliha-srbd) [inline docs/application/web/guides/w3c/supplement/typedarray.md:140]:
```suggestion
	> **Note**
```

**Review by maliha-srbd — COMMENTED** [review]
Reviewed and made some minor changes.

File: docs/application/web/guides/w3c/supplement/typedarray.md (hunk 2 above)
Comment (omar-srbd) [reply docs/application/web/guides/w3c/supplement/typedarray.md:140]: [!NOTE] > **Note**?

Check again.

**Review by maliha-srbd — COMMENTED** [review]
@omar-srbd Please make the changes in lines 106 and 268 for the following:
from "> **Note**" to "> [!NOTE]"

**Comment by omar-srbd** [comment]
@maliha-srbd, I have made changes according to your review.

Ensure all the changes are reflected properly.
Check for any other changes that might have been missed from the reviews.

**Pull request closed without merging**